### ISSUE TITLE
Title & temperature font-size

### ISSUE THREAD

**Issue by anglrus** [bramkragten/weather-card#49]
Hello,
Very good job!
Please, could you tell how it is possible to manage font size for Title and Temp.
It is huge for me :)
I tried to do modification in code but without results.
Thank you
Andrey

**Comment by jjmonteiro** [comment]
Yeah I back this as well, not sure what's the best approach but it definitely doesn't look right on the mobile.
![image](https://user-images.githubusercontent.com/31997649/81820476-92106580-9528-11ea-816f-f4a9e885bb8f.png)

**Comment by bramkragten** [comment]
I've been playing with some options, but haven't found the right one yet.

**Comment by thehaxxa** [comment]
Is this still been looked at?

**Comment by dz0n3s** [comment]
Same problem !

**Comment by DanskerUS** [comment]
Same problem here as well.

**Comment by Malvu** [comment]
Same problem here as well.

**Comment by Shamshala** [comment]
Same as [here](https://github.com/bramkragten/weather-card/issues/47#issuecomment-635208715)

As a temporary fix, install [card-mod](https://github.com/thomasloven/lovelace-card-mod) and try this:
```
entity: weather.home
style:
  .: |
    ha-card div.current span.temp {
      font-size: 4em;
      right: 1em;
      top:0.6em;
     }
    ha-card div.current span.title {
       font-size: 2.7em;
       left: 3em;
       top: 1em;
    }
name: London
type: 'custom:weather-card'
```

**Comment by Malvu** [comment]
thanks, but being a beginner I don't know where to enter this data

**Comment by Shamshala** [comment]
Paste the `style:` part right into your wheather card.

**Comment by dz0n3s** [comment]
Thx,    !

**Comment by Ndrinta** [comment]
+1

**Comment by moccajava** [comment]
+1 changing font size in JS does not effect changes.
